js/sign/src/cli.ts

```diff
@@ -24,10 +26,22 @@ function readOptions() {
     .parse(process.argv);
 }
 
-export function main() {
+async function parseMaybeEncryptedKey(
+  privateKeyFile: string
+): Promise<KeyObject> {
+  try {
+    return parsePemKey(privateKeyFile);
+  } catch (e) {
+    console.warn("This might be an encrypted private key, let's try again.");
+  }
+
+  return parsePemKey(privateKeyFile, await readPassphrase());
+}
+
+export async function main() {
   const options = readOptions();
   const webBundle = fs.readFileSync(options.input);
-  const parsedPrivateKey = parsePemKey(
+  const parsedPrivateKey = await parseMaybeEncryptedKey(
     fs.readFileSync(options.privateKey, 'utf-8')
```

Comment:
Could we change this to read the key as a Buffer (by removing the encoding parameter)? I should've noticed that earlier; treating the private key as raw bytes is better than trying to utf-8 decode it.